docs/reference/ml/ml-shared.asciidoc

```diff
@@ -1427,6 +1427,15 @@ bucket result are deleted from {es}. The default value is null, which means all
 results are retained.
 end::results-retention-days[]
 
+tag::annotations-retention-days[]
+Advanced configuration option. The period of time (in days) that annotations are
+retained. Age is calculated relative to the timestamp of the latest bucket
+result. If this property has a non-null value, once per day at 00:30 (server
+time), annotations that are the specified number of days older than the latest
+bucket result are deleted from {es}. The default value is null, which means all
+annotations are retained.
```

Comment:
Should add a note saying something like "User created annotations are never deleted automatically"

Reply:
Done.